src/pages/Home/index.tsx

```diff
@@ -139,7 +139,7 @@ const useHomeSearchBarStatus = (dispatch: AppDispatch, homeSearchBarVisible: boo
       type: ComponentActions.UpdateHomeSearchBarVisible,
       payload: { homeSearchBarVisible: window.pageYOffset < searchPosition },
     })
-  }, 200)
+  }, 300)
```

Comment:
should we also update the comment above?

Reply:
Reduce the refresh rate to relieve charts pressure

Reply:
Updated https://github.com/nervosnetwork/ckb-explorer-frontend/pull/599/commits/cf23adc7742a78e370e431f40950d8a0412c038e